src/Atc.DotNet/DotnetBuildHelper.cs

```diff
@@ -14,13 +13,16 @@ namespace Atc.DotNet
 {
     public static class DotnetBuildHelper
     {
+        private const int DefaultTimeoutInSec = 1200;
+
         public static Task<Dictionary<string, int>> BuildAndCollectErrors(
             ILogger logger,
```

Comment:
Can we make the ILogger optional ?

Reply:
Yes and new overload-method is added without 'logger'-parameter